kobuki/b166c93.bugzoo.yml

```diff
@@ -0,0 +1,31 @@
+version: '1.0'
+
+blueprints:
+  - type: docker
+    tag: robustrosin/robust:b166c93
+    file: ../Dockerfile
+    context: b166c93
```

Comment:
Here's the problem. `context` specifies the location of the build context, relative to the YAML file. In this case, you could either change context to `.` or omit it entirely (since `.` is the default).

Reply:
Ah. Teaches me to cargo-cult my way through this.

I'll try it.

Reply:
Oh, my mistake. I thought you had put the manifest file into the `b166c93` directory. Your `context` property is correct in this case.

Reply:
No I had the file in `b166c93`. I'll make sure to check before posting again.